cmd/gokr-packer/write.go

```diff
@@ -534,6 +537,30 @@ func writeRoot(f io.WriteSeeker, root *fileInfo) error {
 	return fw.Flush()
 }
 
+func writeRootDeviceFiles(f io.WriteSeeker, rootDeviceFiles []deviceconfig.RootFile) error {
+	kernelDir, err := packer.PackageDir(*kernelPackage)
+	if err != nil {
+		return err
+	}
+
+	for _, rootFile := range rootDeviceFiles {
+		if _, err := f.Seek(rootFile.Offset, io.SeekStart); err != nil {
+			return err
+		}
+
+		fToCopy, err := os.Open(filepath.Join(kernelDir, rootFile.Name))
+		if err != nil {
+			return err
+		}
+		if _, err := io.Copy(f, fToCopy); err != nil {
+			return err
+		}
+		_ = fToCopy.Close()
```

Comment:
Why are you discarding this error? `Close()` (when writing) makes the kernel flush out data, which can well trigger an i/o error that should be reported.

Reply:
This is closing the reader. We've already copied relevant bytes out, which is the important part.
If closing the source file fails, its not fatal - worst case, we end up leaking a file handle for the duration of `gokr-packer`.

Reply:
Ah, I see! Can you rename `fToCopy` to `source`, to make it a little more descriptive?